testgres/node.py

```diff
@@ -640,14 +693,16 @@ def start(self, params=[], wait=True):
         Returns:
             This instance of :class:`.PostgresNode`.
         """
+        if self.is_started:
+            return self
 
         _params = [
-            get_bin_path("pg_ctl"),
-            "-D", self.data_dir,
-            "-l", self.pg_log_file,
-            "-w" if wait else '-W',  # --wait or --no-wait
-            "start"
-        ] + params  # yapf: disable
+                      get_bin_path("pg_ctl"),
```

Comment:
remove excessive tabs/spaces to not override history

Reply:
It doesn't change git history. I don't know why it is showing like this, but no changes here from my side.
![image](https://user-images.githubusercontent.com/5807469/235121651-ea852980-c4bf-47db-9499-ee122ed727bc.png)

Reply:
I fixed tabs according flake8 requirements, I hope it will be ok now